src/VercelEnvVariabler.test.ts

```diff
@@ -209,4 +223,119 @@ describe("VercelEnvVariabler", () => {
         expect(mocked(postEnvVariable)).toHaveBeenCalledTimes(1);
         expect(mocked(patchEnvVariable)).toHaveBeenCalledTimes(2);
     });
+
+    it("Should create ENV_6 for new bar branch and NOT patch ENV_6 for 'old' foo branch", async () => {
+        const variabler = new VercelEnvVariabler(
+            testToken,
+            testProjectName,
+            "ENV_6",
+            testTeamId
+        );
+
+        await variabler.populateExistingEnvVariables();
+        await variabler.processEnvVariables();
+
+        // no patches for (preview) ENV_6 on foo branch...
+        expect(mocked(patchEnvVariable)).not.toHaveBeenCalled();
+        expect(mocked(patchEnvVariable)).not.toHaveBeenCalledWith(
+            expect.anything(),
+            testProjectName,
+            ENV_6_BRANCH_FOO_VARIABLE_ID,
+            expect.objectContaining({
+                value: env6BranchBarValue,
+                target: ["preview"],
+                type: "plain",
+                gitBranch: "bar",
+            })
+        );
+        // ... only a post for (preview) ENV_6 on bar branch
+        expect(mocked(postEnvVariable)).toHaveBeenCalled();
+        expect(mocked(postEnvVariable)).toHaveBeenCalledTimes(1);
+        expect(mocked(postEnvVariable)).toHaveBeenCalledWith(
+            expect.anything(),
+            testProjectName,
+            expect.objectContaining({
+                value: env6BranchBarValue,
+                target: ["preview"],
+                type: "plain",
+                gitBranch: "bar",
+            })
+        );
+    });
+
+    it("Should determine no changes for ENV_6 for foo branch", async () => {
+        // overwrite process.env of ENV_6 for branch bar (set in beforeEach) to old foo branch values (see envVariableFixtures)
```

Comment:
`set in BeforeAll`

Reply:
fixed comment to `... (set in beforeAll)... `